testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/support/support_logging.t.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/**
+ * Tests support logging.
+ */
+StartTest(function(t) {
+  var newLoggerName = 'aaa.example',
+      clickAndWaitForStoreToLoad = function(triggerQuery, storeQuery) {
+        return {
+          waitFor: 'StoresToLoad',
+          args: function() { return t.cq1(storeQuery).getStore(); },
+          trigger: { click: triggerQuery },
+          desc: 'Click "' + triggerQuery + '" and wait for "' +
+                storeQuery + '"\'s store to load'
```

Comment:
At first glance this appears to wait for the store to load and then click, so maybe `waitForStoreToLoadThenClick` is more descriptive? Though I'm not 100% convinced about putting all of this behaviour into a single utility function, especially as it's easy to get the arguments the wrong way round.